changelogs/fragments/534-add-support-for-200gb.yaml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
```

```suggestion
minor_changes:
```

REF: (https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to)

minor_changes
Minor changes to ansible-core, modules, or plugins. This includes new parameters added to modules, or non-breaking behavior changes to existing parameters, such as adding additional values to choices[]. Minor changes are enhancements, not bug fixes. Write in present tense.

bugfixes
Fixes that resolve issues. SHOULD not be used for minor enhancements (use minor_change instead). Use past tense to describe the problem and present tense to describe the fix.